fitbit-react/.gitignore

```diff
@@ -11,6 +11,7 @@
 # production
 
 # misc
+.env
```

Comment:
FYI- you have added .env to gitignore now but still it will be available in git history. There are workarounds for removing but take care next time.

Reply:
Sure👍👍